pkg/labels/labels.go

```diff
@@ -89,6 +89,12 @@ const (
 
 	// PIDContainer is the `nerdctl run --pid` for restarting
 	PIDContainer = Prefix + "pid-container"
+
+	// NerdctlDefaultNetwork indicates whether a network is the default network
+	// created and owned by Nerdctl.
+	// Boolean value which can be parsed with strconv.ParseBool() is required.
+	// (like "nerdctl/default-network=true" or "nerdctl/default-network=false")
+	NerdctlDefaultNetwork = Prefix + "default-network"
```

Comment:
Does this cover multi-network (`nerdctl run --net=foo --net=bar`, commonly invoke via compose) ?

Reply:
I'd also like to see an integration test

Reply:
> Does this cover multi-network (nerdctl run --net=foo --net=bar, commonly invoke via compose)?

The label will only get set on the [default network nerdctl creates](https://github.com/containerd/nerdctl/pull/1538/files#diff-669913e187d4dc726dcb9e01a5666ae0562549545138d25d2c8fc2ceb3e23d30R234), and the label will only be [checked when trying to find the default network](https://github.com/containerd/nerdctl/pull/1538/files#diff-669913e187d4dc726dcb9e01a5666ae0562549545138d25d2c8fc2ceb3e23d30R188).

I'll be adding an integration test shortly.